src/utils/mtev_str.h

```diff
@@ -56,15 +56,12 @@ typedef struct mtev_str_buff{
 API_EXPORT(const char *) strnstrn(const char *, int, const char *, int) __attribute__((deprecated)); /*1.2.9*/
 #endif
 
-#if !defined(BROKEN_MEMMEM) && defined(_GNU_SOURCE)
-#define mtev_memmem(a,b,c,d) memmem(a,b,c,d)
-#else
 API_EXPORT(void *) mtev_memmem(const void *haystack, size_t haystacklen,
                                const void *needle, size_t needlelen);
-#endif
-
```

Comment:
This seems unrelated?

Reply:
Only slightly, but yes.  In the C file, there is the same logic where we call memmem.

I was trying to make sure we don't accidentally squat on what *should be* system stuff.